docs/content/about-iota/iota-architecture/transaction-lifecycle.mdx

```diff
@@ -17,127 +17,107 @@ import ThemedImage from '@theme/ThemedImage';
+2.  **Submit to Validators**: The client application communicates with a full node, which then distributes the transaction to all validators following the IOTA protocol. The validators verify the transaction. If valid, the validators sign it and return the signed transaction to the full node.
 
-3. **Form a Transaction Certificate**: After collecting responses from a supermajority of validators, the client can form a transaction _certificate_. Unlike consensus-based blockchains, IOTA validators are not burdened by needing to propagate signatures or aggregate _certificates_; this is the responsibility of the client or gateway.
+3.  **Form an Aggregate Certificate**: After collecting responses from a supermajority of validators, the full node forms an aggregate transaction _certificate_. Unlike consensus-based blockchains, IOTA validators are not burdened by needing to propagate signatures or aggregate _certificates_; this is the responsibility of the client or gateway.
 
-4. **Send the Certificate**: The client sends the assembled _certificate_ back to all validators. The validators check its validity and acknowledge receipt. If the transaction involves only owned objects, IOTA can process and execute it immediately without waiting for consensus (**fast path consensus**). All _certificates_ are forwarded to the IOTA DAG-based consensus protocol.
+4.  **Send the Certificate**: The client sends the assembled aggregate _certificate_ back to all validators. The validators verify its validity and acknowledge receipt. If the transaction involves only owned objects, IOTA can process and execute it immediately without waiting for consensus (**fast path consensus**). All _certificates_ are forwarded to the IOTA DAG-based consensus protocol.
 
-5. **Consensus and Execution**: The IOTA DAG-based consensus protocol eventually produces a total order of _certificates_. Validators check and execute _certificates_ that involve shared objects.
+5.  **Consensus and Execution**: The IOTA DAG-based consensus protocol eventually produces a total order of _certificates_. Validators verify and execute _certificates_ that involve shared objects.
 
-6. **Effect Certificate**: Clients can gather responses from a supermajority of validators, assemble them into an effect _certificate_, and use it as proof that the transaction is settled.
+6.  **Form an Effect Certificate**: Clients can gather responses from a supermajority of validators, assemble them into an effect _certificate_, and use it as proof that the transaction is settled.
 
-7. **Checkpoints and Reconfiguration**: IOTA forms checkpoints for every consensus commit, which are also used to drive the reconfiguration protocol.
+7.  **Checkpoints and Reconfiguration**: IOTA forms checkpoints for every consensus commit, which are also used to drive the reconfiguration protocol.
 
-## Real-world Example
+## Understanding the Transaction Lifecycle in IOTA
```

Comment:
I'm a little confused about the rest of this page. What is the vision here? We mention variables and stuff. Although this shouldn't be too technical, no?

Reply:
As Vlad mentioned in his review, the `transaction lifecycle` was explained three times, which was somehow confusing to me. I decided to streamline the transaction lifecycle into one simple process and capture concepts that are similar in the three steps. I also removed the code samples to make it not look too technical as you have suspected.